Rx/v2/test/operators/pairwise.cpp

```diff
@@ -21,7 +21,7 @@ SCENARIO("pairwise - enough items to create pairs", "[pairwise][operators]") {
         WHEN("taken pairwise") {
 
             auto res = w.start(
-                [xs, &invoked]() {
```

Comment:
according to visual studio this makes `invoked` decl on line 10 unreferenced. :)